sys/freebsd/pf.txt

```diff
@@ -122,3 +135,20 @@ pfioc_iface {
 	pfiio_size	int32
 	pfiio_flags	int32
 }
+
+pf_ifspeed_v0 {
+	ifname		const[IFNAMSIZ, int8]
+	baudrate	int32
+}
+
+pf_ifspeed_v1 {
+	ifname		const[IFNAMSIZ, int8]
+	baudrate32	int32
+	baudrate	int64
+}
+
+pfioc_states_v2 {
+	ps_len		int32
```

Comment:
Based on the name, is it a length of something? If so, it's better to use `len`/`bytesize` types.